policyengine_us/parameters/gov/states/oh/tax/income/deductions/deductions.yaml

```diff
@@ -2,9 +2,8 @@ description: Ohio deducts these sources from the federal adjusted gross income.
 values:
   2021-01-01:
     - oh_section_179_expense_add_back
-    - qualified_business_income_deduction_person
     - taxable_social_security
-  # - above_the_line_deductions" #only need line 8z
+  #   above_the_line_deductions" #only need line 8z
```

Comment:
Can you make this look more like an informative comment rather than commented out code?